DatadogSDKBridge/Tests/DdRumTests.swift

```diff
@@ -102,8 +102,8 @@ internal class DdRumTests: XCTestCase {
         XCTAssertEqual(mockNativeRUM.receivedAttributes.count, 1)
         let lastAttribtutes = try XCTUnwrap(mockNativeRUM.receivedAttributes.last)
```

Comment:
we maybe can fix a typo in this PR as well: `lastAttribtutes` -> `lastAttributes`. this is in all tests it seems.